package.json

```diff
@@ -12,7 +12,7 @@
     "test": "BABEL_ENV=test ava 'test/**/*.js' --require ./test/helpers/setup.js",
     "test:nyan": "npm test -- --tap | tap-nyan",
     "test:watch": "npm test -- --watch",
-    "cover": "nyc --reporter=text --reporter=html --check-coverage --statements 97.56 --branches 95.92 --functions 98.39 --lines 97.56 npm test",
+    "cover": "nyc --reporter=text --reporter=html --check-coverage --statements 90 --branches 90 --functions 90 --lines 90 npm test",
```

Comment:
While we are changing this, let's only check statements here (so please remove branches, functions, and lines) since Jim confirmed to me the other day that is the only one we are enforcing. Of course we can shoot for higher, but 90% statements should be the low bar.